.github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md

```diff
@@ -12,7 +12,7 @@ Who should invited to [Catena-X NG](https://github.com/catenax-ng)
 GitHub User:
 <!-- Please fill in the GITHUB-USER-ID -->
 
-Vouching person: <!-- Please fill in your Product Owners GITHUB-USER-ID-->
+Vouching person: <!-- Please fill in your GitHub-Teams Maintainer GITHUB-USER-ID or the Product Owners GITHUB-USER-ID -->
```

Comment:
Which org? ;P

Reply:
[chore: added gh-org for gh-team maintainer](https://github.com/eclipse-tractusx/sig-infra/pull/45/commits/daa0a8766c78c9886c7142a695430502fc1ce4a9)